src/explore-education-statistics-common/src/modules/find-statistics/components/SummaryRenderer.tsx

```diff
@@ -73,7 +73,11 @@ export default function SummaryRenderer({
                   {measures[key]}
                   {meta.indicators[key].unit}
                 </p>
-                <p className="govuk-body-s">{dataSummary[index]}</p>
+                {dataSummary ? (
+                  <p className="govuk-body-s">{dataSummary[index]}</p>
+                ) : (
+                  ''
```

Comment:
I think it's better to return `null` if dataSummary isn't defined here, over an empty string.

Also, to conditionally render an element on whether a variable is defined then you can:
`{dataSummary && <p className="govuk-body-s">{dataSummary[index]}</p>}`